Rakefile

```diff
@@ -92,8 +92,18 @@ parsed_files = PARSER_FILES.map do |parser_file|
 end
 
 task "#{path}.gem" => package_parser_files
-desc "Generate all files used racc and kpeg"
-task :generate => parsed_files
+
+desc "Generate all files use racc and kpeg"
+task :generate do
+  unless ENV.key?('BUNDLE_GEMFILE')
+    Gem.install 'racc', '> 1.4.10'
+    Gem.install 'kpeg', '>= 1.3.3'
+  end
+
+  parsed_files.each do |file_name|
+    Rake::Task[file_name].invoke
+  end
+end
```

Comment:
```suggestion
task :generate => [:gem_install, *parsed_files]

task :gem_install do
  unless ENV.key?('BUNDLE_GEMFILE')
    Gem.install 'racc', '> 1.4.10'
    Gem.install 'kpeg', '>= 1.3.3'
  end
end
```

Reply:
This may be better.

```suggestion
task :generate => parsed_files

unless ENV.key?('BUNDLE_GEMFILE')
  task :gem_install do
    Gem.install 'racc', '> 1.4.10'
    Gem.install 'kpeg', '>= 1.3.3'
  end
  file parsed_files => :gem_install
end
```